components/src/core/components/List/List.vue

```diff
@@ -200,8 +200,8 @@ export default defineComponent({
       default: () => ({
         limit: 20 as number,
         perPage: {
-          id: 1,
-          label: '10',
+          id: 2,
```

Comment:
What is the reason to change this from 1 to 2?

Reply:
To change the default pagination to 20.  The normal pagination we use is 20 throughout the system.
If id is not changed here, the highlighted item in the pagination dropdown will be different than what it should highlight